src/reporters/test-case.reporter.js

```diff
@@ -1,5 +1,6 @@
 const WDIOReporter = require("@wdio/reporter").default
 const colors = require("colors");
+const {TraceabilityUtility} = require("../../lib/utils/traceability.util");
```

Comment:
Is there any way to avoid using the full paths by using barrels, just like we do in imports (e.g. import { IconClass } from '@e2e-model';?)

Just asking, as I have used such things just for imports, so far

Reply:
Imports between from javascript files should be done at least for now as it is.